plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -626,4 +632,17 @@ private static void validateOrcBloomFilterColumns(ConnectorTableMetadata tableMe
             throw new TrinoException(INVALID_TABLE_PROPERTY, format("Orc bloom filter columns %s not present in schema", Sets.difference(ImmutableSet.copyOf(orcBloomFilterColumns), allColumns)));
         }
     }
+
+    public static TableScan buildTableScan(Table icebergTable, MetadataTableType metadataTableType)
+    {
+        return createMetadataTableInstance(icebergTable, metadataTableType)
+                .newScan();
+    }
+
+    public static Map<String, Integer> columnNameToPositionInSchema(Schema schema)
+    {
+        return mapWithIndex(schema.columns().stream(),
+                (column, position) -> Maps.immutableEntry(column.name(), position))
+                .collect(Collectors.toMap(e -> e.getKey(), e -> e.getValue().intValue()));
```

Comment:
Style nit: prefer `toImmutableMap` and use functional syntax for `getKey`
```suggestion
        return mapWithIndex(schema.columns().stream(),
                (column, position) -> Maps.immutableEntry(column.name(), position))
                .collect(toImmutableMap(Entry::getKey, e -> e.getValue().intValue()));
```

Reply:
updated.